static/src/javascripts/projects/common/utils/raven.js

```diff
@@ -46,8 +46,11 @@ define([
                     }
                 }
 
-                return config.switches.enableSentryReporting &&
+                return adBlockerOn.then(function(adBlockerOn){
```

Comment:
This change is the _very_ problematic. It requires changes in raven.js, which is a third-party library.